cmake/TrMacros.cmake

```diff
@@ -159,6 +159,13 @@ macro(tr_add_external_auto_library ID DIRNAME LIBNAME)
         set(${ID}_LIBRARIES ${${ID}_LIBRARY})
 
         set(${ID}_EXT_PROJ_CMAKE_ARGS)
+
+        list(APPEND ${ID}_EXT_PROJ_CMAKE_ARGS
+            "-DCMAKE_SYSTEM_NAME=${CMAKE_SYSTEM_NAME}"
+            "-DCMAKE_SYSTEM_VERSION=${CMAKE_SYSTEM_VERSION}"
+            "-DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}"
+            "-DCMAKE_FIND_ROOT_PATH=${CMAKE_FIND_ROOT_PATH}")
```

Comment:
I'd expect these variables to be set by the toolchain file which is already passed through down below, so it should be safe to remove them.